src/RawHtmlFieldInput.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+import {bindActionCreators} from 'redux';
+import {connect} from 'react-redux';
+import {changeBlockValue} from './actions';
+import {
+  getFieldName,
+  isStatic,
+  shouldRunInnerScripts
+} from './processing/utils';
+
+
+const MutationObserver = window.MutationObserver
+                      || window.WebKitMutationObserver
+                      || window.MozMutationObserver;
+
+
+@connect(null, (dispatch, props) => {
+  const {fieldId, blockId} = props;
+  return bindActionCreators({
+    changeBlockValue: value => changeBlockValue(fieldId, blockId, value),
+  }, dispatch);
+})
```

Comment:
This already seems to be done in `FieldInput`. Could it pass its `changeBlockValue` definition to `RawHTMLFieldInput`, instead of also connecting this component to the store?